sds_data_manager/lambda_code/SDSCode/batch_starter.py

```diff
@@ -476,20 +472,20 @@ def lambda_handler(event: dict, context):
 
         # Start Batch Job execution for each instrument
         for downstream_data in downstream_instruments_to_process:
-            filename = downstream_data["filename"]
-
-            command = downstream_data["prepared_data"]
+            command = downstream_data["command"]
             logger.info(f"Submitting job with this command - {command}")
             # NOTE: The batch job name should contain only
             # alphanumeric characters and hyphens.
-            job_name = f"{instrument}-{level}-job"
+            level_to_process = command[3]
```

Comment:
Any way to change this to where it isn't a hard-coded int?

Reply:
Not for this because we need to send batch command as list of string. We could think about this better way to do this down the road.